### PR TITLE
feat(CX-2828): change empty graph copy

### PR DESCRIPTION
**Pull request by lordkiz**
### Description
- PR updates the copy when we have no data for the selected timeframe.

<p>

</p>

This PR resolves [CX-2828]

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Update empty graph copy - kizito

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2828]: https://artsyproduct.atlassian.net/browse/CX-2828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx [hunk 1]
```diff
@@ -74,15 +78,21 @@ export const MedianSalePriceChart: React.FC = () => {
...
               >
-                <Text textAlign="center" variant="sm" color="black60">
-                  Incomplete Data for the selected medium
+                <Text variant="sm" color="black60">
+                  {hasInsights
+                    ? "Incomplete data for the selected medium"
+                    : "No data currently available"}
                 </Text>
+                {!hasInsights && (
+                  <Text variant="sm" color="black60">
```
Comment (MounirDhahri) [inline src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx:92]: **thought**: I believe design would like to have the text centered here, unless something changed, if so please disregard my comment

Comment (lordkiz) [reply src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx:92]: Yeah but design has not handled the possibility of 2 blocks of texts. It ended up looking like this:

-==============-
-------======------
-==============-
-------======------

which looked ugly compared to:

-=============-
-===============-

File: src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx [hunk 2]
```diff
@@ -42,6 +42,10 @@ export const MedianSalePriceChart: React.FC = () => {
     return formatted
   }
 
+  const hasInsights = !!categories.length
+
+  const PADDING_LEFT_AND_RIGHT = 80
```
Comment (MrSltun) [inline src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx:47]: Suggestion: 👀 
```suggestion
  const PADDING_HORIZONTAL = 80
```

File: src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx:92]: aha trueeee. good point, thank you for the visualization 🙏

**Review by MounirDhahri — APPROVED** [review]
LGTM. Thanks for addressing my comment 🙏

**Review by MrSltun — APPROVED** [review]
Looks good to me! Thank you Kizito!! 🌟 
I just left a small suggestion, but feel free to merge without it :)

File: src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx (hunk 2 above)
Comment (lordkiz) [reply src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx:47]: Hmmm. PADDING_LEFT_AND_RIGHT is insinuated from the possible padding a graph would have. 20 is our padding horizontal. And the graph will add another 20 (thereabouts) to allow for y-axis labels. So it is not a true paddingHorizontal. We just need these assumptions to calculate the left value for absolute positioned text.

**Pull request merged**